Documentation/Hosting/AmazonGameLift/GameLift.md

```diff
@@ -0,0 +1,105 @@
+# MultiplayerSample Project with Amazon GameLift
+
+This README covers testing and running MultiplayerSample with Amazon GameLift.
```

Comment:
Link to GameLift as a service, call out this is entirely optional and describe what GameLift does.

ie


```suggestion
This README covers optional setup, testing and running on [Amazon GameLift](https://aws.amazon.com/gamelift/), an AWS service to make hosting and scaling game servers easier. 
```

Reply:
Fixed

Reply:
Thank you! Added your changes